generators/app/templates/client/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015", "react", "stage-0"],
+  "plugins": ["react-hot-loader/babel"]
+}
```

Comment:
I don't like the idea of having babel's configuration on the `.babelrc` file because it further decentralizes configuration. But that is just me.